web/src/utils/overview-panels.ts

```diff
@@ -48,11 +48,11 @@ export const getDefaultOverviewPanels = (): OverviewPanel[] => {
     { id: 'top_dropped_state_donut', isSelected: true },
     { id: 'top_dropped_cause_donut', isSelected: true },
     { id: 'top_dropped_bar_total', isSelected: true },
-    { id: 'top_avg_dns_latency_donut', isSelected: false },
-    { id: 'top_dns_rcode_donut', isSelected: false },
-    { id: 'top_dns_rcode_bar_total', isSelected: false },
-    { id: 'top_avg_rtt_donut', isSelected: false },
-    { id: 'top_avg_rtt_line', isSelected: false }
+    { id: 'top_avg_dns_latency_donut', isSelected: true },
```

Comment:
should we be more selective and not display them all by default? E.g. show donuts, which are more lightweight, than line/bar charts for the same metric ?

Reply:
I guess it's more a matter of advertising what's available here more that anything don't you think ?

That selection is kept in cache so this will affect only the first time until the user actually configure the panels (or get new panels not yet assigned from new feature / operator update).